### PR TITLE
Fix: Event consumer Airdrop claim events

### PR DESCRIPTION
**Pull request by rajeevbbqq**
1: Fix Airdrop event consumer
2: Separated Slack channel name to config.py

### CODE REVIEW

File: airdrop/application/services/airdrop_services.py [hunk 1]
```diff
@@ -57,9 +60,11 @@ def airdrop_listen_to_events(self, event):
 
     def update_airdrop_window_claim_status(self, event):
         try:
-            event_payload = event['json_str']
+            event_payload = json.loads(event["json_str"])
+            event_payload = ast.literal_eval(event_payload)
+            user_address = event_payload['claimer']
```
Comment (anandrgitnirman) [inline airdrop/application/services/airdrop_services.py:65]: Why is this line twice ?

**Review by anandrgitnirman — CHANGES_REQUESTED** [review]
Please raise seprate PR For slack alert and for the event consumer, 
can you also please provide details on the PR for the consumer on how it worked in Ropsten , was amount  not processed becuase we had int , but were getting bigint ?

**Comment by anandrgitnirman** [comment]
All the build is failing , please check

File: airdrop/application/services/airdrop_services.py (hunk 1 above)
Comment (rajeevbbqq) [reply airdrop/application/services/airdrop_services.py]: Fixed in the latest code

**Review by anandrgitnirman — APPROVED** [review]

**Pull request merged**